.circleci/config.yml

```diff
@@ -603,7 +603,7 @@ jobs:
             fi
       - run-yarn-command:
           command-name: Create static visualization js bundle
-          command: build-static-viz
+          command: yarn build-static-viz
```

Comment:
That kindof defeats the purpose of abstracting it in `run-yarn-command` (might as well just use the regular `run`).

Reply:
Yeah, agree, but soon we will be moving the e2e to GHA then I will remove completely the command.